middleware/route-guard.js

```diff
@@ -0,0 +1,10 @@
+const isLoggedIn = (req, res, next) => {
+  if (!req.session.user) res.redirect("/login");
+  next();
+};
+
+const isLoggedOut = (req, res) => {
```

Comment:
Here, you are missing the `next()` that would prevent the website to be stuck on the middleware.
```
const isLoggedOut = (req, res, next) => {
  if (req.session.user) res.redirect('/');
  next();
};
```